sql/moz-fx-data-shared-prod/telemetry/firefox_android_clients/view.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.firefox_android_clients`
+AS
+SELECT
+  * EXCEPT (adjust_network, install_source),
+  CASE
+  WHEN
+    adjust_network NOT IN (
+      '',
```

Comment:
Can we normalize the empty and null Adjust networks? Not sure if they mean e.g. Adjust unavailable, or unknown attribution, etc.

Reply:
@bani how should we normalize this value?
Specially because it's different scenarios, NULL = not received and empty = received without any value.

Reply:
In the context that this table will be used for data incidents, not marketing attribution, It's ok to group empty and null into a single value. I think `Unknown` is a good value for the purpose of Looker visualizations.

Reply:
Thank you. So I've normalized in the view and we have the raw data in the table for the scenario where data is used by Marketing.

Reply:
@bani can you describe what you mean by "data incidents"?

Reply:
@fbertsch data incidents is when one or more metrics we care about deviate from the expected values and we start getting questions "why", and then we start the investigation by segmenting the metric in various ways. The idea from this table was a follow up from [this incident](https://docs.google.com/document/d/170I6yaaSws8LJEsMmXqDQ0U4IChVmRmyfgXmIhGUkrs/edit?usp=sharing)

Reply:
For reference, I'd add that in addition to analysis of incidents, this table was extended from the original requirement to add more value:

- Enhanced the existing Firefox Android new_profile ping so that it can be used for clients first attributes and not require the metrics ping anymore to retrieve attribution details.
- Use this structure as a baseline to create new_profile pings for apps that don't have one yet, specifically Firefox Desktop.
- Integrate additional "first seen" dimensions to enable further analysis: first seen country, ISP, device manufacturer, device model.

And it is also a reason to keep the raw data in the table, and implement the normalization and calculations in the view.

Reply:
Good point!